### ISSUE TITLE
Blurry fonts on new label engine

### ISSUE THREAD

**Issue by 25hash** [grokability/snipe-it#13656]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

After updating to 6.2.0 wit the new label engine, the font is blurry on the label. On screen the font are sharp but when printed they're not. Tried from Google Chrome and Safari, same result.

### Reproduction steps

1. Generate a label for any asset with the new label engine activated
2. Preview the label without any visual imperfection
3. Print the label

### Expected behavior

The printed label should be the same quality as the preview.

### Screenshots

Browser preview :
![SCR-20230926-jafa](https://github.com/snipe/snipe-it/assets/3055793/edff1186-3dec-4d5a-b716-9b4b7b49489e)

Printed result : 
![IMG_6137](https://github.com/snipe/snipe-it/assets/3055793/c92bd7a6-6027-41a2-95ba-53d60920645b)

### Snipe-IT Version

6.2.0

### Operating System

Ubuntu

### Web Server

Official Docker Image (Apache2 2.4.52)

### PHP Version

Official Docker Image (8.1.2)

### Browser

Chrome and Safari

**Comment by snipe** [comment]
IIRC, people have said lowering the resolution on the printer resolves this issue.

**Comment by 25hash** [comment]
> IIRC, people have said lowering the resolution on the printer resolves this issue.

Same font problem, but with a lower quality. I'm using a Dymo LabelWriter 450 if that changes anything.
The new label engine is a welcome feature anyway, i wasn't able to get any label to work on safari and now it's working for that part ! 

![IMG_6138 jpg](https://github.com/snipe/snipe-it/assets/3055793/b04c39ea-6e23-4676-894e-6a5e233948ca)

On the right from Chrome and left Safari

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by 25hash** [comment]
> Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?
> 
> :) Thanks!

Yes it's solved, sorry I fogot to close it. The issue was the browsers. Changed to Firefox and everything's printing fine. Thank you so much for the follow up, you're the best ! :)

**Comment by snipe** [comment]
Thanks for letting us know!
